docs/guides/03-interactivity-popups.md

```diff
@@ -0,0 +1,325 @@
+
+```js
+await countriesLayer.setPopupClick([
+  {
+    attr: 'name',
+    title: null 
+  },
+  {
+    attr: 'pop_est',
+    title: 'Estimated Population',
+    format: value => Math.round((value / 1.0e+6) * 100) / 100 + 'M inhabitants'
+  },
+  {
+    attr: 'economy',
+    title: 'Economy Group'
+  },
+  {
+    attr: 'income_grp',
+    title: 'Income Group'
+  }
+]);
+```
+
+<div class="example-map">
+    <iframe
+        id="interactivity-popups-step-6"
+        src="/developers/web-sdk/examples/maps/guides/interactivity-popups/step-6.html"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+</div>
+> View this step [here](/developers/web-sdk/examples/maps/guides/interactivity-popups/step-6.html)
+
+Once we have added a pop-up for a layer, we can easily disable it by calling the pop-up function with a null argument.
+
+```js
+await countriesLayer.setPopupHover(null);
+await countriesLayer.setPopupClick(null);
+```
+
+### All together
+
+This is the complete example with feature highlighting and pop-ups both for hover and click events on the features.
+
+<div class="example-map">
+    <iframe
+        id="getting-started-step-7"
+        src="/developers/web-sdk/examples/maps/guides/interactivity-popups/step-7.html"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+</div>
+
+> You can explore the final step [here](/developers/web-sdk/examples/maps/guides/interactivity-popups/step-7.html)
+
+```html
+<!DOCTYPE html>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Interactivity and Pop-ups Guide - Step 7</title>
+
+    <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.css' rel='stylesheet' />
+  
+    <link href="https://libs.cartocdn.com/airship-style/v2.4.0/airship.css" rel="stylesheet" >
+
+    <style>
+      body {
+        margin: 0;
+        padding: 0;
+      }
+
+      #map {
+        width: 100vw;
+        height: 100vh;
+      }
+    </style>
+
+  </head>
+
+  <body>
+
+    <div id="map"></div>
+
+    <!-- Include deck.gl from unpkg CDN -->
+    <script src="https://unpkg.com/deck.gl@8.2.0/dist.min.js"></script>
+
+    <!-- Include Mapbox GL from the Mabpox CDN -->
+    <script src='https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.js'></script>
+
+    <!-- Include Web SDK from the CARTO CDN -->
+    <script src="https://libs.cartocdn.com/web-sdk/v1.0.0-alpha/index.min.js"></script>
+
+    <!-- Include Airship from the CARTO CDN -->
+    <script type="module" src="https://libs.cartocdn.com/airship-components/v2.4.0/airship/airship.esm.js"></script>
+    <script nomodule="" src="https://libs.cartocdn.com/airship-components/v2.4.0/airship/airship.js"></script>
```

Comment:
Nice addition! We should add that in other airship uses in our examples, as chrome suggests